### ISSUE TITLE
Documentation utilisateur pour les liens deux à deux

### ISSUE THREAD

**Issue by nsenave** [InseeFr/Pogues#694]
Actuellement, il n'y a pas de documentation pour les liens deux à deux :

- sur la page d'aide accessible depuis Pogues : https://inseefr.github.io/Pogues/fr/guide-utilisateur/guide-utilisateur.html
- dans la documentation Bowie : https://inseefr.github.io/Bowie/pogues/tutoriel/

NB : il faudrait peut-être refondre l'une ou l'autre (et mettre un renvoi vers la seule doc résultante)

NB 2 : en termes d'UX le composant est difficile à appréhender tel qu'il est actuellement (amélioration UX => doc plus légère)

**Comment by romaintailhurat** [comment]
Indeed, still on my todo list, but with a low priority, our only user being the TCM designer. But still, it needs to be done. 😃 📝

**Comment by AnneHuSKa** [comment]
https://inseefr.github.io/Pogues/fr/guide-utilisateur/guide-utilisateur.html : xpath donc pas de liens 2 à 2
https://inseefr.github.io/Bowie/pogues/Le%20guide/16-liens-2a2/
